app.rb

```diff
@@ -0,0 +1,62 @@
+require "sinatra/base"
+require "sinatra/activerecord"
+require "bcrypt"
+
+require_relative "lib/peep"
+require_relative "lib/user"
+
+
+class Chitter < Sinatra::Base
+  register Sinatra::ActiveRecordExtension
+  enable :sessions
+
+  get "/" do
+    @peeps = Peep.all.order(created_at: :desc)
+    if session[:session_user_id]
+      @user = User.find(session[:session_user_id])
+    end
+    erb :index
+  end
+
+  post "/new_peep/:id" do
```

Comment:
I think the convention of creating a new resource in CRUD / REST apps is POST request to `/peeps` or` /users` etc. And then you POST path can redirect to `/user/:id` or `/peep/:id` once it has created it (or back to the submission page with error if not created)